CODE_OF_CONDUCT.md

```diff
@@ -38,114 +23,106 @@ All participants in our in-person events and online communications are expected
 To make clear what is expected, everyone participating in activities associated with the Turing Way project is required to conform to this Code of Conduct.
 This Code of Conduct applies to all spaces managed by the Turing Way project including, but not limited to, in-person focus groups and workshops, and communications online via GitHub.
 
-The lead investigator of the Turing Way - Dr Kirstie Whitaker - is responsible for enforcing the Code of Conduct.
-She can be contacted by emailing [kwhitaker@turing.ac.uk](mailto:kwhitaker@turing.ac.uk).
-You may contact [Anna Krystalli](mailto:a.krystalli@sheffield.ac.uk) or [Ben Murton](mailto:bmurton@turing.ac.uk) if you would prefer not to contact Dr Whitaker.
+### Contact for Reporting
+
+The co-lead-investigator of _The Turing Way_ - Dr. Kirstie Whitaker ([kwhitaker@turing.ac.uk](mailto:kwhitaker@turing.ac.uk)) and Dr. Malvika Sharan ([msharan@turing.ac.uk](mailto:msharan@turing.ac.uk)) - are responsible for enforcing the Code of Conduct.
+
+If you prefer not to contact the project leads, you may also contact [Anna Krystalli](mailto:a.krystalli@sheffield.ac.uk), who is our ombudsperson.
+You can reach out to our Research Community Manager, Anne Lee Steele ([asteele@turing.ac.uk](mailto:asteele@turing.ac.uk)) and Research Project Manager, Alexandra Araujo Alvarez ([aaraujo.alvarez@turing.ac.uk](mailto:aaraujo.alvarez@turing.ac.uk)) for support in making a report to the project leads on your behalf or anonymously.
```

Comment:
Anonymously probably isn't the right word here if someone is reaching out to specific contact emails, right?

```suggestion
You can reach out to our Research Community Manager, Anne Lee Steele ([asteele@turing.ac.uk](mailto:asteele@turing.ac.uk)) and Research Project Manager, Alexandra Araujo Alvarez ([aaraujo.alvarez@turing.ac.uk](mailto:aaraujo.alvarez@turing.ac.uk)) for support in making a report to the project leads on your behalf, with or without your name associated with it.
```

Reply:
@aleesteele and @AlexandraAAJ can you please confirm that you understand your roles here?

Reply:
This looks good to me.

Reply:
Looks good to me too. Thanks @malvikasharan. Definitely would be interested in doing training regarding how to respond to CoC's going forward.

Reply:
Thanks, Anne. I suggest reading this for the role described here: https://malvikasharan.github.io/blogs/coc-facilitators/. I can look for my availability in the last quarter of the year for the Turing Way-specific training.